src/app/components/local/store-details/store-details.component.ts

```diff
@@ -148,4 +169,30 @@ export class StoreDetailsComponent implements OnInit {
       return a + b.price;
     }, 0);
   }
+
+  plusSlides(n:any) {
+    this.showSlides(this.slideIndex += n);
+  }
+  
+   currentSlide(n:any) {
```

Comment:
Add access modifiers for all the methods & properties. ( public if it is used outside of the component class & private if it is used only within the class)